changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,131 @@
+      <column name="scheduleId" type="INTEGER" autoIncrement="true">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="schedulesId_pkey" />
+      </column>
+      <column name="settlementId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="completed" type="TIMESTAMP" />
+      <column name="planned" type="TIMESTAMP" />
+      <column name="started" type="TIMESTAMP" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="7">
+    <addForeignKeyConstraint baseColumnNames="settlementId" baseTableName="schedules" constraintName="fk_settlements_settlementId" referencedColumnNames="settlementId" referencedTableName="settlements" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="8">
+    <createTable tableName="organisations">
+      <column name="sbi" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="organisationsId_pkey" />
+      </column>
+      <column name="address" type="VARCHAR(255)" />
+      <column name="emailAddress" type="VARCHAR(127)" />
+      <column name="frn" type="BIGINT" />
+      <column name="name" type="VARCHAR(127)" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="9">
+    <createTable tableName="claims">
+      <column name="calculationId" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="claimsId_pkey" />
+      </column>
+      <column name="paymentRequestId" type="INTEGER" >
+        <constraints nullable="true" />
+      </column>
+      <column name="sbi" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="agreementNumber" type="VARCHAR(20)" /> <!-- diff to paymentRequests'? -->
+      <column name="calculationDate" type="TIMESTAMP" />
+      <column name="invoiceNumber" type="VARCHAR(20)" />
+    </createTable>
+  </changeSet>
+  <changeSet author="Marc Templeton" id="10">
+    <addForeignKeyConstraint baseColumnNames="paymentRequestId" baseTableName="claims" constraintName="fk_paymentRequests_paymentRequestId" referencedColumnNames="paymentRequestId" referencedTableName="paymentRequests" />
+    <addForeignKeyConstraint baseColumnNames="sbi" baseTableName="claims" constraintName="fk_organisations_sbi" referencedColumnNames="sbi" referencedTableName="organisations" />
+  </changeSet>
+  <changeSet author="Marc Templeton" id="11">
+    <createTable tableName="schemes">
+      <column name="schemeCode" type="VARCHAR(10)">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="schemesId_pkey" />
+      </column>
+      <column name="calculationId" type="INTEGER" >
+        <constraints nullable="false" />
+      </column>
+      <column name="areaClaimed" type="DECIMAL(11,5)" /> <!-- based on largest land ownership over ~500,000 arces: https://whoownsengland.org/2020/10/06/mapping-the-habitats-of-englands-ten-largest-institutional-landowners/ -->
```

Comment:
Surely not needing to this precise and large, right?

Reply:
I _believe_ the area is stored to four decimal places.

I'm sure this is big enough, though checking the Cross Border database that handles BPS payments for the largest landowners, that is storing `19, 4` whereas most values seemed to have been originally `18,4`.  I wonder if there is a reason the land areas were bumped.

We could go with `19,4` to be safe and also see if we can find out why `19,4` was chosen